src/hub/Hub.sol

```diff
@@ -225,31 +229,13 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
     }
 
     /// @inheritdoc IHub
-    function sendTriggerUpdateHoldingAmount(
-        PoolId poolId,
-        ShareClassId scId,
-        AssetId assetId,
-        address who,
-        uint128 amount,
-        bool isIncrease
-    ) public payable {
-        _isManagerAndPaid(poolId);
-
-        sender.sendTriggerUpdateHoldingAmount(poolId, scId, assetId, who, amount, isIncrease);
-    }
-
-    /// @inheritdoc IHub
-    function sendTriggerUpdateShares(
-        uint16 centrifugeId,
-        PoolId poolId,
-        ShareClassId scId,
-        address who,
-        uint128 shares,
-        bool isIssuance
-    ) public payable {
+    function sendTriggerIssueShares(uint16 centrifugeId, PoolId poolId, ShareClassId scId, address who, uint128 shares)
```

Comment:
post-morten. The API should be just: `triggerIssueShares`?

Reply:
Seems inconsistent with all the other functions having `send`? But also fine with me, then let's do for all the trigger messages. Do you want to put up a PR?

Reply:
Yeah, I can do it